tests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/basic/OpenNewApplicationWizardWithNoProjectTest.java

```diff
@@ -164,6 +169,43 @@ public void testOpenNewApplicationWizardFromOpenShiftExplorerWithNoProjects() {
 		closeWizard();
 	}
 	
+	@Test
+	public void testOpenNewApplicationWizardFromOpenShiftExplorerWithNoProjectsAndCancelNewProject() {
+		LogView logView = new LogView();
+		logView.open();
+		logView.deleteLog();
+
+		OpenShiftExplorerView explorer = new OpenShiftExplorerView();
+		explorer.reopen();
+		
+		OpenShift3Connection connection = 
+				explorer.getOpenShift3Connection();
+		connection.select();
+		new ContextMenu(OpenShiftLabel.ContextMenu.NEW_OS3_APPLICATION).select();;
+		
+		try {
+			new WaitWhile(new ShellWithTextIsAvailable(OpenShiftLabel.Shell.CREATE_OS_PROJECT), TimePeriod.LONG);
```

Comment:
That sounds quite reasonable, but since I copied it from another test method testOpenNewApplicationWizardFromOpenShiftExplorerWithNoProjects(), can it mean that it is a coding error there?